media_kit_video/lib/src/video/video_web.dart

```diff
@@ -460,7 +462,7 @@ Future<void> defaultEnterNativeFullscreen() async {
 /// Makes the native window exit fullscreen.
 Future<void> defaultExitNativeFullscreen() async {
   try {
-    document.exitFullscreen();
+    await document.exitFullscreen().toDart;
```

Comment:
Does this work?

Reply:
just tested and yes it works on javascript code (cant compile to wasm since the player is not migrated yet but if it works for javascript it should work for wasm)